docs/developer-center/guides/04-Quickstart.md

```diff
@@ -317,501 +306,1821 @@ Map([
 
 There they are! To learn more about creating isochrones and isodistances check out the [Location Data Services guide](/developers/cartoframes/guides/Location-Data-Services).
 
+> Note: You will see how to publish a map in the last section. If you already want to publish this map, you can do it by calling `map.publish('starbucks_isochrones', password=None)`.
+
 
 ### Enrich your data with demographic data
 
 Now that you have the area of influence calculated for each store, let's take a look at how to augment the result with population information to help better understand a store's average revenue per person.
 
-First, let's find the demographic variable we need. We will use the `Catalog` class that can be filter by country and category. In our case, we have to look for USA demographics datasets. Let's check which geographies (spatial resolution) are available.
+> Note: To be able to use the Enrichment functions you need an enterprise CARTO account with Data Observatory 2.0 enabled. Contact your CSM or contact us at sales@carto.com for more information about this.
```

Comment:
```suggestion
> Note: To be able to use the Enrichment functions you need an enterprise CARTO account with Data Observatory 2.0 enabled. Contact your CSM or [contact us](mailto:sales@carto.com) for more information about this.
```

Reply:
I will include this with a small change :)